samples/apps/copilot-chat-app/webapp/src/libs/semantic-kernel/skMultiUserChat.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
+import * as signalR from "@microsoft/signalr";
+
+export class SKMultiUserChat {
+    private hubConnection!: signalR.HubConnection;
+
+    constructor(private readonly serviceUrl: string) {}
+
+    // Set up a SignalR connection to the specified hub URL
+    setupSignalRConnectionToChatHub() {
+      const connectionHubUrl = (new URL('/chatHub', this.serviceUrl)).toString();
+
+      const signalRConnectionOptions = {
+        skipNegotiation: true,
```

Comment:
@Vivihung, @adrianwyatt I can't establish a connection without skipping negotiations with the server. This is being blocked by the CORS policy of the app. I am not familiar enough with CORS to know if that has some unintended consequences?

Error:
Ensure CORS requests include credentials only when allowed
A cross-origin resource sharing (CORS) request was blocked because it was configured to include credentials but the Access-Control-Allow-Credentials response header of the request or the associated preflight request was not set to true.

Reply:
@glahaye  - do you have any experience here?